express-mock-af/bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
```

Comment:
please delete the spaces after all comments

Reply:
Not mine... I wont adjust the files of him